cmds/core/wc/wc.go

```diff
@@ -92,8 +92,18 @@ func command(stdin io.Reader, stdout io.Writer, stderr io.Writer, p params, args
 	}
 }
 
+// TODO: if we change the behavior to return an error, then inputting multiple
+// non-existing files will result in a premature exit and the behavior will
+// differ from the original implementation. This should be adjusted. Until then
+// disable the noparam linter to fail because we always return nil.
+//
+//nolint:unparam
 func (c *cmd) run() error {
```

Comment:
This is the wrong fix. 
What should be done is to use errors.Join
so this command and the nolint: actually make things worse. 
This is why I am so concerned about this PR. It is guiding people to do the wrong thing.